### ISSUE TITLE
CLI cron.php: Call to undefined function JavaScriptEscape() in Subs.php:2184

### ISSUE THREAD

**Issue by m4z** [SimpleMachines/SMF#6102]
#### Description
I'm recently updated my forum to the GH version from this weekend, and my `cron`-triggered `cron.php` fails since:
```
PHP Fatal error:  Call to undefined function JavaScriptEscape() in /home/[…]/public_html/Sources/Subs.php on line 2184
```

### Steps to reproduce
1. Run `php […]/cron.php`
2. Notice you don't receivce forum notifications anymore, and check the error mails of the account running the forum ;)

### Environment (complete as necessary)
- Version/Git revision: GH version from 2020-05-02
- Database Type: postgres but probably any
- Database Version: 9.4-ish
- PHP Version: 5.6-ish

**Comment by Sesquipedalian** [comment]
Ah, yes, I see the problem. Solution is either (A) to move the `JavaScriptEscape()` function from QueryString.php to Subs.php, or (B) to include QueryString.php from cron.php.

In general, I’d lean toward option A in a situation like this, but the ExportProfileData background task has to load QueryString.php anyway, so I’ll probably go with option B.

Either way, a fix should be forthcoming later today.

**Comment by Sesquipedalian** [comment]
A fix for this has been added to #6101

**Comment by m4z** [comment]
Thanks, you're awesome! Will test once merged.
